x/crosschain/keeper/evm_hooks.go

```diff
@@ -327,7 +328,7 @@ func ValidateZrc20WithdrawEvent(event *zrc20.ZRC20Withdrawal, chainID int64) err
 	// The event was parsed; that means the user has deposited tokens to the contract.
 
 	if chains.IsBitcoinChain(chainID) {
-		if event.Value.Cmp(big.NewInt(0)) <= 0 {
+		if event.Value.Cmp(big.NewInt(constant.BTCWithdrawalDustAmount)) < 0 {
 			return fmt.Errorf("ParseZRC20WithdrawalEvent: invalid amount %s", event.Value.String())
```

Comment:
Let's add `constant.BTCWithdrawalDustAmount` to the error message to make it more helpful

Reply:
https://github.com/zeta-chain/node/pull/2362/commits/7b122ebee5b2067759cf95fd2594becd0cb7a576